src/components/calltree/CallTreeEmptyReasons.js

```diff
@@ -5,10 +5,11 @@
 
 import React, { PureComponent } from 'react';
 
-import EmptyReasons from '../shared/EmptyReasons';
-import { selectedThreadSelectors } from '../../selectors/per-thread';
+import EmptyReasons from 'firefox-profiler/components/shared/EmptyReasons';
+import { selectedThreadSelectors } from 'firefox-profiler/selectors/per-thread';
 import { oneLine } from 'common-tags';
-import explicitConnect, { type ConnectedProps } from '../../utils/connect';
+import type { ConnectedProps } from 'firefox-profiler/utils/connect';
+import explicitConnect from 'firefox-profiler/utils/connect';
```

Comment:
You don't need to split these imports actually, you can still keep them in one import statement.